packages/object-entries/index.d.ts

```diff
@@ -1,2 +1,2 @@
 export default function entries<T extends any[]>(obj: T): [number, T[number]][];
-export default function entries<T extends object>(obj: T): [keyof T, T[keyof T]][];
\ No newline at end of file
+export default function entries<T extends Record<keyof any, unknown>>(obj: T): [keyof T, T[keyof T]][];
```

Comment:
FTI: `keyof any` is equivalent to `string | number | symbol`, which represents the types that can be keys of an object.